server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md

```diff
@@ -1,20 +1,51 @@
+# Tailscale Funnel 
+**Requirements**
+
+- Enable [HTTPS certificates](https://login.tailscale.com/admin/dns) in the [admin console](https://login.tailscale.com/admin/dns).
+- [Add Funnel to Policy](https://login.tailscale.com/admin/acls) in the admin console.
+
 ---
-description: >-
```

Comment:
I think you should keep some sort of description directly under the first header to keep the format the same as the other guides. It gives some context about what prerequisites there are to use this guide (e.g. having a Tailscale account). The distinction of this being an option that doesn't involve port-forwarding is also important.